src/monitor/src/main/java/ee/ria/xroad/monitor/CertificateInfoSensor.java

```diff
@@ -85,7 +83,6 @@ public CertificateInfoSensor() throws Exception {
                 .addExtractor(CertificateType.SECURITY_SERVER_TLS, new InternalTlsExtractor())
                 .addExtractor(CertificateType.AUTH_OR_SIGN, new TokenExtractor());
 
-        updateOrRegisterData(new JmxStringifiedData());
```

Comment:
why was this removed, and does it alter behaviour if metrics data is fetched before INITIAL_DELAY?

Reply:
I didn't return "empty" initialisations but for other two sensor they are initialised. Also tested that update missing from constructor doesn't break monitor request.